src/membership.md

```diff
@@ -36,18 +36,22 @@ Lang team members can propose new additions to the team as follows:
 These are the questions we ask ourselves when deciding whether someone would be a good choice as a lang team member.
 
 * Has this person demonstrated **strong language design skills**?
-    * Have they led an impactful initiative to completion?
-    * Have they identified a criticial flaw in someone else's design that led to the design being altered?
+    * Have they made major contributions to an area of the language, such as leading an impactful initiative to completion?
```

Comment:
```suggestion
    * Have they made notable contributions to an area of the language, such as leading an impactful initiative to completion?
```

Or perhaps some other word would work better; the previous text just said "led an impactful initiative", but calling it "major" feels like a further qualification. I think we'd value someone who had an impact in an area, even if that area itself wasn't "major". The important thing is a chance to see their design and shepherding skills in action contributing to the language.